packages/integration-tests/src/helper.ts

```diff
@@ -0,0 +1,89 @@
+import { execSync } from "child_process";
+import { ethers } from "ethers";
+import { promises as fs } from "fs";
+import * as path from "path";
+import * as request from "supertest";
+import { Provider } from "zksync-web3";
+
+import { environment, localConfig } from "./config";
+import { Logger } from "./entities";
+
+import type { BaseProvider } from "@ethersproject/providers/src.ts/base-provider";
+
+export class Helper {
+  async txHashLogger(txType: string, txValue: string, tokenName?: string) {
+    const logMessage = `TxHash for ${txType} ${Logger.textSeparator} ${txValue}`;
+
+    if (tokenName === undefined) {
+      return console.log(logMessage);
+    } else {
+      return console.log(logMessage, ` ${tokenName}`);
+    }
+  }
+
+  async executeScript(script: string) {
+    const output = execSync(script, { encoding: "utf-8" });
```

Comment:
Please use `exec` and wrap it with Promise, `execSync` is blocking.

Reply:
updated